src/modules/assets/copy/copy.ts

```diff
@@ -0,0 +1,43 @@
+import { type IOdsModulesCopy } from './IOdsModulesCopy';
+
+export const enCopy: IOdsModulesCopy = {
+    addressInput: {
+        clear: 'Clear',
+        paste: 'Paste',
+    },
+    assetDataListItemStructure: {
+        unknown: 'Unknown',
+    },
+    memberDataListItemStructure: {
+        yourDelegate: 'Your Delegate',
+        you: 'You',
+        delegations: 'Delegations',
+        votingPower: 'Voting Power',
+    },
+    approvalThresholdResult: {
+        stage: 'Stage',
+        of: 'of',
+        members: 'Members',
+    },
+    majorityVotingResult: {
+        winningOption: 'Winning Option',
+        stage: 'Stage',
+    },
+    proposalDataListItemStatus: {
+        youHaveVoted: "You've voted",
+    },
+    proposalDataListItemStructure: {
+        by: 'By',
+        creatorsNumber: (count) => `${count}+ ${count > 1 ? 'creators' : 'creator'}`,
```

Comment:
We probably don't need to handle a singular case here, `creatorsNumber` should actually be `maxCreatorsNumber` and there's no case where `count` should be set to `0`. Even `0+` might still be more than one 😄

Reply:
✅ https://github.com/aragon/ods/pull/221/commits/5442722929ba987b81593c7ae4ddbf400604f23a